package.json

```diff
@@ -9,7 +9,7 @@
     "start": "cross-env NODE_ENV=development electron .",
     "lint": "standard",
     "test": "cross-env NODE_ENV=test mocha test/unit/**/*.spec.js",
-    "test:e2e": "xvfb-maybe cross-env NODE_ENV=test LIBP2P_ALLOW_WEAK_RSA_KEYS=1 mocha test/e2e/**/*.e2e.js --exit",
+    "test:e2e": "xvfb-maybe cross-env NODE_ENV=test playwright test test/e2e/launch.e2e.test.js",
```

Comment:
:information_source:  switched to runner native to playright (just to avoid any bugs caused by legacy mocha stack)